plugins/rebalance/api/resp.go

```diff
@@ -0,0 +1,68 @@
+package rebalance
```

Comment:
As this file contains both request and response structs, you can name this file as `req_resp.go`